drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/importers/loaders/esg_funding_assistance_loader.rb

```diff
@@ -0,0 +1,115 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# matriculation to new platform
+# creates CustomService and CustomDataElements
+module HmisExternalApis::AcHmis::Importers::Loaders
+  class EsgFundingAssistanceLoader < SingleFileLoader
+    def perform
+      records = build_records
+      # destroy existing records and re-import
+      model_class.where(data_source: data_source).destroy_all if clobber
```

Comment:
@eanders @ttoomey how can we bypass acts_as_paranoid here, so that records actually get wiped rather than marked as deleted?

I just realized this clobber is just marking records as deleted. Given we're planning to do this migration in prod 3X, we don't want those older delete records to be present in the db at all.

Reply:
If it needs to bypass the `destroy` hooks, I _think_ thats OK. We need to clobber most of the Custom Data Elements before running this anyway.

Reply:
You can call `really_destroy!`
https://github.com/rubysherpas/paranoia#usage

Reply:
OK, I think we should do that.

Reply:
Probably [delete_all!](https://github.com/ActsAsParanoid/acts_as_paranoid#real-deletion) is the way to go for for a record set as long as there's no dependents